lib/app/widgets/bottom_navigation.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:go_router/go_router.dart';
+import 'package:gruene_app/app/constants/bottom_navigation_items.dart';
+import 'package:gruene_app/app/constants/theme_colors.dart';
+import 'package:gruene_app/app/constants/theme_text_styles.dart';
+import 'package:gruene_app/i18n/translations.g.dart';
+
+class BottomNavigation extends StatefulWidget {
+  const BottomNavigation({super.key});
+
+  @override
+  BottomNavigationState createState() => BottomNavigationState();
+}
+
+class BottomNavigationState extends State<BottomNavigation> {
+  int _currentIndex = 0;
+
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    final currentRoute = GoRouterState.of(context).path;
+    final index = bottomNavigationItems.indexWhere((item) => item.route == currentRoute);
+    if (index != -1) {
+      _currentIndex = index;
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final t = Translations.of(context);
+    return SizedBox(
+      height: 64.0,
+      child: BottomNavigationBar(
```

Comment:
:thought_balloon: There is also a new [NavigationBar](https://api.flutter.dev/flutter/material/NavigationBar-class.html) which is "preferred for new applications and applications that are configured for Material 3" (see [BottomNavigationBar](https://api.flutter.dev/flutter/material/BottomNavigationBar-class.html)).


However, since the click dummy does not look like it uses material design v3, we probably should stick with the `BottomNavigationBar` (see the video [here](https://api.flutter.dev/flutter/material/NavigationBar-class.html)). What do you think?

Reply:
No, I would definitely agree that we should follow the Figma style and since that does not use material v3, we should probably stick to `BottomNavigationBar`. Just wanted to mention it if you think we should inquire or change anything.